src/network/MissionControlProtocol.cpp

```diff
@@ -185,24 +185,31 @@ void MissionControlProtocol::handleJointPowerRequest(const json& j) {
 }
 
 void MissionControlProtocol::sendRoverPos() {
-	auto heading = robot::readIMUHeading();
-	auto gps = robot::readGPS();
-	if (gps.isValid() && heading.isValid()) {
-		double orientX = 0.0;
-		double orientY = 0.0;
-		double orientZ = std::sin(heading.getData() / 2);
-		double orientW = std::cos(heading.getData() / 2);
-		double posX = gps.getData()[0];
-		double posY = gps.getData()[1];
+	auto imu = robot::readIMU();
+	auto gps = gps::readGPSCoords();
+	log(LOG_DEBUG, "imu=%d, gps=%d\n", imu.isValid(), gps.isValid());
```

Comment:
It might be better to put `imu_valid` and `gps_valid` instead of just `imu` and `gps` so that you know what its indicating. 

You can also do ```log(LOG_DEBUG, "imu_valid=%s, gps=%s\n", imu.isValid() ? "true" : "false", gps.isValid() ? "true" : "false")``` which will give you true and false instead of 1 and 0.

So in total you  would get `imu_valid=true, gps_valid=true`. Which is a bit more clear than `imu=1, gps=1`